db/dcrpg/queries.go

```diff
@@ -3278,6 +3278,7 @@ func appendChartBlocks(charts *cache.ChartData, rows *sql.Rows) error {
 			// datasets sized properly.
 			bigwork = big.NewInt(int64(blocks.Chainwork[len(blocks.Chainwork)-1]))
 		}
+		blocks.Height = append(blocks.Height, height)
```

Comment:
```golang
	SelectBlockStats = `SELECT height, size, time, chainwork, numtx
		FROM blocks
		WHERE is_mainchain
		AND height > $1
		ORDER BY height;`
```
This `ORDER BY height` condition makes it unnecessary to have `blocks.height` cached data.

Reply:
It's helpful for the sanity check here: https://github.com/decred/dcrdata/pull/1426/files/2eb2d445c52be6fc94fcb7f0b79d17606aaf28a8#diff-87f7a577b6af45d0284e4c3fbb98ddf5R3292
But I get what you mean about implicit ordering and not using a slice of heights when plotting.

Reply:
Migwi mentioned that the heights were implicit.  Still, I like filling in the heights slice, since we have it.  It might be useful for debugging at some point, such as if the slice lengths do not agree with the height, as might happen if there's a missing datapoint in the DB somewhere.

Reply:
I've bumped the cache version from 4 -> 5 since v4 cache gobs will have an empty `Heights` slice and fail validation